src/code.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+#
+
+# The following line causes bash to exit at any point if there is any error
+# and to output each line as it is executed -- useful for debugging
+set -e -x -o pipefail
+
+#
+# Download inputs
+#
+dx download "$bedfile"
+
+to_remove="data.bed"
+#echo $bamfile_name | sed -e "s/$to_remove$//"
+pannum=$(echo $bamfile_name | sed "s/$to_remove//")
+
+
+#pannum=${$bamfile_name%$to_remove}
+echo $pannum
+#pannum="${pannum}"
+
+#read the api key as a variable
+API_KEY=$(cat '/home/dnanexus/auth_key')
+
+
+
+#make test dir
+mkdir to_test
+cd to_test
+echo $project_name
+dx download $project_name:output/*$pannum*001.ba* --auth $API_KEY
+filecount="$(ls *"$pannum"* | grep . -c)"
+if (( $filecount < 6 )); then
```

Comment:
Will RPKM generate solid number with just the reference samples? Do we / do we need to take into account the sex of the samples for X genes?